crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -636,7 +636,7 @@ impl RpcServerConfig {
 #[derive(Debug, Clone, Default, Eq, PartialEq)]
 pub struct TransportRpcModuleConfig {
     /// http module configuration
-    http: Option<RpcModuleSelection>,
+    pub http: Option<RpcModuleSelection>,
```

Comment:
@gakonst @mattsse I want to create some tests, just make field pub or add query method?

Reply:
add method

Reply:
yes, please add functions instead.